pages/specific-words-and-phrases.md

```diff
@@ -8,23 +8,34 @@ alphabetical order. The bold term shows the accepted form
 (capitalization, hyphenation, punctuation), with accompanying text
 explaining usage. Examples appear in italics.
 
--   18F employees are “18F team members” or “18F staffers.” Avoid all iterations of “18F-r.”
+-   18F employees are *18F team members* or *18F staffers*. Avoid all iterations of “18F-r.”
 
 **A**
 
+-   **ages** - avoid hyphens in ages unless it clarifies the text. For example, *a group of 9 18-year-old White House tourists*
```

Comment:
In keeping with the numbers section below, maybe "nine 18-year-olds"? I don't think it undermines the point even in word then numeral construction.